usagov/vote-gov-drupal
# Vote 843 -> install reporter

**_Delete all details that do not apply to this PR!_**

## Jira ticket (required)
Insert Jira ticket link
https://bixal-projects.atlassian.net/jira/software/c/projects/VOTE/boards/254?modal=detail&selectedIssue=VOTE-843&assignee=611272e0eb7aef00694b8a97

## Description (optional)

This Pr will add testing reporting to the project to allow us to have better access to trouble shooting and to share reports to team and clients 

## Deployment and testing (required, if applicable)
### Testing steps
1. pull down the branch 
2. start up local 
3. cd into `testing`
4. run `npm i`
5. run `npm run cy:test`
6. check in the reports folder there should be an index.html file and then can open that in your default browser to view the report
